src/Resources/Resources/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Added support for creating and setting Template Spec in Bicep file [#15313]
```

Comment:
I think **update** makes more sense here because `Set-` cmdlet is used to "Modifies a Templace Spec".

```suggestion
* Added support for creating and updating Template Spec in Bicep file [#15313]
```